Test Automation Course/src/topic6/Main.java

```diff
@@ -0,0 +1,16 @@
+package topic6;
+
+public class Main {
+
+    public static void main(String[] args) {
+
+        ContulPersoanei cont1 = new ContulPersoanei("Olaru Vladislav",565678,50000);
```

Comment:
Sugerez sa construiesti un account cu date invalide si sa prinzi exceptia intr-un try catch, in catch deja sa pui date valide